widgets/pagination.js

```diff
@@ -1,6 +1,8 @@
 var React = require('react');
 
 var utils = require('../lib/utils.js');
+var autoHide = require('../decorators/autoHide');
+var Pagination = autoHide(require('../components/Pagination/Pagination.js'));
```

Comment:
Maybe it's time to get rid of this rule ;P (that would be to much indeed) Ok let's not do that then :)